### DISCUSSION TITLE
My school put the blocker in their network

### DISCUSSION THREAD

**Discussion by HeroLink6** [3kh0/ext-remover#898] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I don't know what to do, but I corrupted my blocker extension, and the network still blocks any tabs I want to go to. Is there any way I can fix this?

**Comment by HeroLink6** [comment]
(Bookmarklets are also disabled)

**Reply by v4pl** [reply]
Use Skiovox and then install vpn like hola vpn thats what i did.

**Reply by HeroLink6** [reply]
Skiovox is blocked for me now... and I don't have a phone.

**Comment by brobcoolguy** [comment]
mobile hotspot?

**Reply by HeroLink6** [reply]
Nope

**Comment by CJBrandi** [comment]
vpn?

**Comment by lolpc100** [comment, marked as answer]
it needs skiovox but you can use browser.rammerhead.org

**Comment by ilovetoucans** [comment]
if you are on a personal account ublock origin should help
